docker/py3-linter-dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.5-alpine
+
+RUN pip install yapf==0.16.0 && pip install flake8==3.3.0 && pip install pytest==3.0.6
```

Comment:
Not sure why we install pytest in this container.  Reading `tasks.py`, it looks like this container's only job is to run yapf and flake8?